code/midtown/mmcity/cullcity.cpp

```diff
@@ -85,3 +98,162 @@ void mmCullCity::Cull()
         }
     }
 }
+
+static agiMeshCardVertex RainMeshCard[4] {
+    {-0.1f, 2.0f, 0.4f, 1.0f},
+    {-0.1f, -2.0f, 0.4f, 0.0f},
+    {0.1f, -2.0f, 0.6f, 0.0f},
+    {0.1f, 2.0f, 0.6f, 1.0f},
+};
+
+void mmCullCity::Init(char* name, asCamera* camera)
+{
+    char city_path[64];
+    char lm_path[64];
+
+    IsSnowing = (MMSTATE.Weather == mmWeather::Snow);
+    CityName = arts_strdup(name);
+
+    arts_strcpy(city_path, name);
+    arts_strcat(city_path, "city");
+
+    arts_strcpy(lm_path, name);
+    arts_strcat(lm_path, "lm");
+
+    TEXSHEET.allow_remapping_ = (MMSTATE.TimeOfDay == mmTimeOfDay::Sunset or MMSTATE.TimeOfDay == mmTimeOfDay::Night);
+
+    switch (MMSTATE.Weather)
+    {
+        case mmWeather::Rain: TextureSuffix = "_fall"_xconst; break;
+        case mmWeather::Snow: TextureSuffix = "_win"_xconst; break;
+        default: TextureSuffix = nullptr; break;
+    }
+
+    StaticLog = arts_fopen(arts_formatf<64>("%s_static.csv", name), "w");
+
+    Loader()->BeginTask(LOC_STRING(MM_IDS_LOADING_CITY_LAYOUT), 0.0f);
+
+    DLPTemplate* city_dlp = nullptr;
+    DLPTemplate* lm_dlp = nullptr;
+
+    if (DevelopmentMode)
+    {
+        city_dlp = GetDLPTemplate(city_path);
+        if (!city_dlp)
+            Quitf("Unable to load city '%s'", name);
```

Comment:
Can you comment out these two lines? There's a patch in patches.cpp which removes this `Quitf`